tests/jeos/glibc_locale.pm

```diff
@@ -209,7 +208,6 @@ sub run {
     select_console('root-console');
     ensure_serialdev_permissions;
     (test_users_locale($rc_lc_reverted, $test_data_lang{$lang_ref}) eq $original_glibc_string) or die "Locale has changed after reboot!\n";
-    reset_consoles;
 }
```

Comment:
Why are you removing this part when in case of sle12sp5 it returns in L#201?

Reply:
ready for review

Reply:
> Please remove, not needed, `reset_consoles` on line 213, it's done in `test_users_locale` also in wait_boot.

we have decide to deal with this module later. The purpose of this PR is to enable initial testing for sle12sp5

Reply:
one or the other :) i prefer @dzedzo's as the modules works now! its up to you guys

Reply:
Sure, I don't mind it's not big deal.

Reply:
ready for review

Reply:
@dzedro

> It was my idea to solve http://dzedro.suse.cz/tests/18763 this issue http://dzedro.suse.cz/tests/18759#step/curl_https/2
Where user-console is considered active but not being active and failing.

That failure is because of a broken needle. `tty4-selected` must not have the `user-console` tag.

Reply:
True, I didn't consider needle being wrong.